main.go

```
@@ -0,0 +1,317 @@
+func generateSummaryReport(items []ProjectItem, filename string) error {
+	file, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	totalBounty := 0.0
+	for _, item := range items {
+		if item.BountyAmount != "" {
+			bountyValue := 0.0
+			fmt.Sscanf(item.BountyAmount, "%f", &bountyValue)
```

golangci-lint (1.64.8)

Error return value of `fmt.Sscanf` is not checked

(errcheck)